spec/requests/reading_list_items_spec.rb

```diff
@@ -22,11 +22,13 @@
     it "returns archives item if no param" do
       put "/reading_list_items/#{reaction.id}"
       expect(reaction.reload.status).to eq("archived")
+      expect(user.last_reacted_at).not_to eq(user.reload.last_reacted_at)
```

Comment:
Rather than relying on ActiveRecord to local memoize this here, I think a cleaner approach would be.
```ruby
original_reacted_at = user.last_reacted_at
put "/reading_list_items/#{reaction.id}"
expect(reaction.reload.status).to eq("archived")
expect(original_reacted_at).not_to eq(user.reload.last_reacted_at)
```
Same with the example below.

Reply:
We could also use an actual `change` assertion here:

```ruby
expect do 
  put "/reading_list_items/#{reaction.id}"
end.to change { user.reload.last_reacted_at }
expect(reaction.reload.status).to eq("archived")
```

Reply:
I have never thought of being able to use `change` by giving `put "..."` to `expect` as a block. Thank you!